### ISSUE TITLE
run when close gui runs twice

### ISSUE THREAD

**Issue by melinstagibson** [Tuke-Nuke/TuSKe#9]
I mentioned this in discord but who knows if you ever saw it

The run when close gui always runs twice, thus duplicating the items and sending the message twice.
Im running the lastest paperspigot 1.12, bensku dev26, latest version of skquery (lime) tuske, shaprsk, skstuff, wolvsk, skellett and skutilities 

on inventory close doesnt cause this error btw
```
command /test:
    trigger:
        create a gui with virtual chest with 4 rows named "         &1&lArena & Duels":
            run when close gui:
                loop all items in  gui-inventory:
                    make player drop loop-item
                send "%gui-player% %gui-inventory% %name of gui-inventory%"
        open last created gui to player
```

**Comment by Tuke-Nuke** [comment]
Do you have any plugin for protocol support like ViaVersion or ProtocolSupport or something?

**Comment by Tuke-Nuke** [comment]
Found the issue, it was a huge mistake while I was testing some stuffs.

**Comment by Tuke-Nuke** [comment]
Fixed in [1.8.1](https://github.com/Tuke-Nuke/TuSKe/releases/tag/v1.8.1)
